src/components/common/SpotButton.tsx

```diff
@@ -0,0 +1,43 @@
+import { SpotButtonProps } from '@/types/CategoryField'
+import Image from 'next/image'
+
+const buttonInfo = {
+  air: {
+    classList: 'bottom-[60px] right-[1px]',
+    alt: '대기 오염',
+  },
+  energy: {
+    classList: 'top-[23px] right-[21px]',
+    alt: '에너지 절약',
+  },
+  recycle: {
+    classList: 'top-[2px] left-[2px]',
+    alt: '분리수거',
+  },
+  soil: {
+    classList: 'top-[77px] left-[61px]',
+    alt: '토양 오염',
+  },
+  warming: {
+    classList: '-top-[38px] left-[84px]',
+    alt: '지구온난화',
+  },
+  water: {
+    classList: 'bottom-[37px] left-[19px]',
+    alt: '수질 오염',
+  },
+}
+export default function SpotButton(props: SpotButtonProps) {
+  const { name, status, onClick } = props
```

Comment:
아마 여기 props에서 `isClickable`을 구조분해할당하지 않아서 잊은 것 같은데.. 이 경우 디폴트값으로 false를 주면 해결 될 것 같아요
```suggestion
  const { name, status, onClick, isClickable = false } = props
```

Reply:
구현부 작성 부탁드리겠습니당